scom/stackstate_checks/scom/scom.py

```diff
@@ -92,23 +110,30 @@ def getComponentDetails(self, id, session, scom_ip, domain, username, password):
             list.update({str(detail.get("name").encode('utf-8')):str(detail.get("value").encode('utf-8'))})
         return list
 
-    def nextNodes(self, serviceid, session, scom_ip, domain, username, password, system_type):
+    def nextNodes(self, serviceid, session, scom_ip, domain, username, password, system_type,integration_mode):
         ids = []
         response = (session.get('http://' + scom_ip + '/OperationsManager/data/objectInformation/' + serviceid,
                                 auth=HttpNtlmAuth(domain + '\\' + username, password))).json()
         data = response.get("relatedObjects", [])
         for idd in data:
             ids.append(idd.get("id"))
+            type_data = {
+                            "criteria": "(Id = '"+str(idd.get("id"))+"')"
+                            }
+            type_response = (session.post('http://'+ scom_ip +'/OperationsManager/data/scomObjects',
```

Comment:
It is much better to use the `base_url` that includes the protocol. You should then create a function to creates a valid SCOM URL.. without duplication of slashes!

Reply:
indeed that the perfect scenario. However, there is a bug in stacksatate with stackpack url field contains "/" . if you enter "/" in the url field and the stackpack creates a view. You end up with totally empty view, even if you try to filter anything within this view, you do not get any components. That's why i am trying to avoid using "/" in the url